.env-docker-compose

```diff
@@ -0,0 +1,70 @@
+#########################################################vvv
```

Comment:
Instead of having a file you could put the environment variables in the `docker-compose` file. This might get confused with the `.env` file which is completely different

Reply:
Yes, I'm not satisfied with this yet.